### ISSUE TITLE
Allow init command to run per cluster

### ISSUE THREAD

**Issue by anders-elastisys** [elastisys/compliantkubernetes-apps#1777]
**Is your feature request related to a problem? Please describe.**
Similar to [this](https://github.com/elastisys/compliantkubernetes-apps/issues/1472), we would like to be able to run the `init` script for each cluster separately.

**Describe the solution you'd like**
Should be able to specify `wc` or `sc` when running `init` command to only initialize for the specified cluster, i.e:
```
./bin/ck8s init wc
./bin/ck8s init sc
```

**Describe alternatives you've considered**

**Additional context**

**Definition of done:**
It is possible to run `init` command per cluster (`sc` or `wc`).

**Comment by anders-elastisys** [comment]
Putting this issue on hold, while waiting on [another PR](https://github.com/elastisys/compliantkubernetes-apps/pull/1778) to be merged, as it requires similar changes.
